Jenkinsfile

```diff
@@ -0,0 +1,61 @@
+pipeline {
```

Comment:
❓ Would it be worth adding a high-level comment on `pipeline` to explain what it's for and what it does?